targets/python/source/source/PlayFabHTTP.py

```diff
@@ -0,0 +1,36 @@
+import json
+import PlayFabSettings
+import PlayFabErrors
+import requests
+
+def DoPost(urlPath, request, authType, authKey, extraHeaders):
```

Comment:
All usage of postProcessingFunc in the API file is exactly the same in all cases.
We should make DoPost accept postProcessingFunc, and do all that work inside of DoPost to avoid that huge mess of copy-paste in the API template.